File: JDK11/opentraceability/src/main/java/opentraceability/models/common/Certificate.java
package opentraceability.models.common;

import opentraceability.Constants;
import opentraceability.utility.attributes.OpenTraceabilityAttribute;
import opentraceability.utility.attributes.OpenTraceabilityJsonAttribute;
import java.time.OffsetDateTime;

public class Certificate {
    @OpenTraceabilityJsonAttribute(name="gdst:certificationType")
    @OpenTraceabilityAttribute(ns = Constants.GDST_NAMESPACE, name="certificationType")
    public String certificateType = null;

    @OpenTraceabilityAttribute(ns = "", name="certificationAgency")
    public String agency = null;

    @OpenTraceabilityAttribute(ns = "", name="certificationStandard")
    public String standard = null;

    @OpenTraceabilityAttribute(ns = "", name="certificationValue")
    public String value = null;

    @OpenTraceabilityAttribute(ns = "", name="certificationIdentification")
    public String identification = null;

    @OpenTraceabilityAttribute(ns = "", name="certificationStartDate")
    public OffsetDateTime startDate = null;

    @OpenTraceabilityAttribute(ns = "", name="certificationEndDate")
    public OffsetDateTime endDate = null;
}

File: Kotlin/OpenTraceability/src/main/kotlin/opentraceability/models/common/certificate.kt
package opentraceability.models.common

import opentraceability.Constants
import opentraceability.utility.attributes.*
import java.time.OffsetDateTime

class Certificate {

    @OpenTraceabilityJsonAttribute("gdst:certificationType")
    @OpenTraceabilityAttribute(Constants.GDST_NAMESPACE, "certificationType")
    var certificateType: String? = null

    @OpenTraceabilityAttribute("", "certificationAgency")
    var agency: String? = null

    @OpenTraceabilityAttribute("", "certificationStandard")
    var standard: String? = null

    @OpenTraceabilityAttribute("","certificationValue")
    var value: String? = null

    @OpenTraceabilityAttribute("","certificationIdentification")
    var identification: String? = null

    @OpenTraceabilityAttribute("","certificationStartDate")
    var startDate: OffsetDateTime? = null

    @OpenTraceabilityAttribute("","certificationEndDate")
    var endDate: OffsetDateTime? = null

}
